src/core/server/saved_objects/saved_objects_config.ts

```diff
@@ -12,6 +12,7 @@ import type { ServiceConfigDescriptor } from '../internal_types';
 const migrationSchema = schema.object({
   batchSize: schema.number({ defaultValue: 1_000 }),
   maxBatchSizeBytes: schema.byteSize({ defaultValue: '100mb' }), // 100mb is the default http.max_content_length Elasticsearch config value
+  ignoreUnknownObjects: schema.maybe(schema.string()),
```

Comment:
I think we should add a custom validation function here to make sure this is following the correct format

Reply:
Do you think it is worth creating a new SemanticVersion type in @kbn/config-schema ?

Reply:
If you want to take a shot a it, please feel free to do so. Can be done as a follow-up though as it's fairly isolated. A custom validation function would be fine too in the meantime.